src/resources/Resources.js

```diff
@@ -0,0 +1,41 @@
+export const CHEF_RESOURCES = {
+    NAME: 'Name',
+    DESCRIPTION: 'Description',
+    RESTAURANTS: 'Restaurants',
+    IMAGE: 'Image',
+  };
+  
+  export const DISH_RESOURCES = {
+    NAME: 'Name',
+    PRICE: 'Price',
+    RESTAURANT: 'Restaurant',
+    IMAGE: 'Image',
+    TAGS: 'Tags',
+    DESCRIPTION: 'Description',
+  };
+  
+  export const RESTAURANT_RESOURCES = {
+    NAME: 'Name',
+    IMAGE: 'Image',
+    RATING: 'Rating',
+    CHEF_ID: 'Chef ID',
+    CHEF_NAME: 'Chef Name',
+    DISHES: 'Dishes',
+  };
+
+  export const ICON_RESOURCES = {
+    CREATE: 'Create',
+    CREATE_NEW: 'Create New',
+    EDIT: 'Edit',
+    DELETE: 'Delete',
+    IMAGE: 'Image',
+    CHEF: 'Chef',
+    CANCEL: 'Cancel',
+    SAVE: 'Save',
+    DETAILS: 'Details',
+    ID: 'ID',
+    NAME: 'Name',
+    ACTION: 'Action'
+  };
+
+  
```

Comment:
Let's include it in the same JSON.
Also, if the model's resources use something generic like 'Name' or 'Save', these should be placed in general resources.

'ICON_RESOURCES' is not a suitable name. All the content is general and not specific.